tests/unit/services/observer-manager-test.js

```diff
@@ -0,0 +1,41 @@
+import { module, test } from 'qunit';
+import { setupTest } from 'ember-qunit';
+import sinon from 'sinon';
+
+module('Unit | Service | observer-manager', function(hooks) {
+  setupTest(hooks);
+
+  hooks.beforeEach(function() {
+    this.observerManager = this.owner.lookup('service:ember-scroll-modifiers@observer-manager');
+
+    sinon.stub(this.observerManager._admin, 'observe');
+    sinon.stub(this.observerManager._admin, 'unobserve');
+    sinon.stub(this.observerManager._admin, 'addEnterCallback');
+    sinon.stub(this.observerManager._admin, 'addExitCallback');
+
+    this.options = {
+      root: undefined,
+    }
+
+    this.callback = () => {};
+  });
+
+  test('API', function(assert) {
```

Comment:
Please make these separate tests instead of cramming everything into one test. This should be 4 tests.